src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/CommandHandler/IdentityCommandHandler.php

```diff
@@ -76,6 +77,7 @@
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Identity\Command\VetSecondFactorCommand;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Identity\CommandHandler\Exception\DuplicateIdentityException;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Identity\Service\RegistrationMailService;
+use function var_export;
```

Comment:
Dev left-over?

Reply:
`Use function` actually serves a purpose in terms of performance optimization;
https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/